src/App.css

```diff
@@ -8,41 +10,78 @@ body {
   display: flex;
   justify-content: center;
   align-items: center;
-  background-color: rgb(235, 225, 225);
+  background-color: #f5f6fa;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+.main-container {
+  width: 87.5vw;
+  height: 50.5vw;
+  margin: 9.3vw 6.063vw 6.063vw 6.063vw;
+  border-radius: 4px;
+  box-shadow: 0 0 60px 0 rgba(0, 0, 0, 0.05);
+  background-color: white;
```

Comment:
- [ ] You have done well implementing all the previous changes, good job 💪🏽 🥳 . However, it seems that the new changes have introduced a bug as the background colours are overlapping as shown below, please fix it

### Your design

<img width="1265" alt="Screenshot 2023-07-08 at 22 26 33" src="https://github.com/JuanDBta/bookstore-project/assets/64774936/6c43f651-b9b7-4de8-abc3-d46aaa42d239">

### Zeplin design
![Screenshot 2023-07-08 at 22 26 33 (2)](https://github.com/JuanDBta/bookstore-project/assets/64774936/8e1769d7-9054-4a0c-be54-a6e7d3043fd4)

Reply:
Hi @Ghiftee...bug fixed!! May I ask you to check it out please?? <img src="https://media1.giphy.com/media/MFIsOqzodLr7ewnkUb/giphy.gif"/>

Reply:
Hello @JuanDBta, it is great now
Good job 👏🏽 👍🏽

Reply:
<img src="https://media0.giphy.com/media/tsX3YMWYzDPjAARfeg/giphy.gif"/> Great! Thanks for the review!